TECHMANIA/Assets/Scripts/Paths.cs

```diff
@@ -386,4 +384,44 @@ public static bool IsInStreamingAssets(string path)
         return absolutePath;
     }
     #endregion
+
+#if UNITY_ANDROID
+    #region Android Permissions
+    // Check whether Android R has permission to read non media files (.tech, .json, etc) or not.
+    public static bool IsAndroidExternalStorageManager ()
```

Comment:
`IsAndroidExternalStorageManager`, `AskForAndroidPermissions` and `HasAndroidStoragePermissions` have little to do with Paths. I would put them into a separate Android utility class.